ui/common/src/commonMain/kotlin/social/androiddev/common/composables/UserAvatar.kt

```diff
@@ -26,10 +27,9 @@ fun UserAvatar(
     url: String
 ) {
     AsyncImage(
-        load = { loadImageIntoPainter(url = url) },
-        painterFor = { remember { it } },
+        url = url,
         contentDescription = "User avatar",
-        modifier = modifier.width(48.dp).clip(RoundedCornerShape(5.dp))
+        modifier = modifier.width(48.dp).height(48.dp).clip(RoundedCornerShape(5.dp))
```

Comment:
modifier.size(48.dp) should already set it for both dimensions

Reply:
for some reason it was not rendering the avatars after switching to the new library. We can play with it later but yeah it never assumed a height